Workshops.qmd

```diff
@@ -26,12 +26,21 @@ You will learn the basics of the R programming language for data wrangling in th
 ![](img/moritz-ketzer.png){width="100"} ![](img/aaron-peikert.png){width="100"} Monday 11 September, 14:30-16:30
 
 For selected participants who already know the basics of R, there will be the opportunity to learn the basics of the Julia programming language through a facilitated self-paced tutorial.
```

Comment:
Only for R-users or also for experienced Fortran-Programmers preferring to learn julia over R? ;) (see my comment below)

Reply:
I haven't written that part. Malika did. I think most of the other workshops require knowledge of R.